pkg/virt-operator/creation/components/webhooks.go

```diff
@@ -228,6 +230,29 @@ func NewVirtAPIValidatingWebhookConfiguration(installNamespace string) *v1beta1.
 					},
 				},
 			},
+			{
+				Name: "virt-launcher-eviction-interceptor.kubevirt.io",
+				// We don't want to block evictions in the cluster in a case where this webhook is down.
+				// The eviction of virt-launcher will still be protected by our pdb until this webhook is up again.
+				FailurePolicy: &failurePolicy,
```

Comment:
shouldn't we be using ignore on failure here? It looks like the failure policy it to fail

Like your comment points out, this will mean that all evictions are blocked across the cluster if our component is down.